scalafmt.sh

```diff
@@ -4,20 +4,24 @@ set -e
 
 BOOTSTRAP_DIR="$HOME/.scalafmt"
 SCALAFMT_VERSION="1.4.0"
+SCALAFMT_ORG="com.geirsson:scalafmt-cli_2.12"
```

Comment:
A bit misleading - it's not really just the organisation name

Reply:
That's true, suggestions for a better name? Splitting to org and scala version separately seems like overkill

Reply:
`ModuleId`?)